Warmup-1/intMax/test/index.test.ts

```diff
@@ -0,0 +1,23 @@
+// Given three int values, a b c, return the largest.
+
+import { assert } from "chai";
+import { intMax } from "../src";
+
+// intMax(1, 2, 3) → 3
+// intMax(1, 3, 2) → 3
+// intMax(3, 2, 1) → 3
+
+describe("intMax", () => {
+  it("should return the largest value 3 when passed 1,2,3", () => {
+    const result = intMax(1, 2, 3);
+    assert.equal(result, 3);
+  });
+  it("should return the largest value 3 when passed 1,3,2", () => {
+    const result = intMax(1, 3, 2);
+    assert.equal(result, 3);
+  });
+  it("should return the largest value 3 when passed 3,2,1", () => {
```

Comment:
This seems like an incorrect test name?

Reply:
I don't see anything wrong with the test name, but i do see a defect in the test data and I have corrected the test data.

Reply:
The test name dictates what the test does, so if the test does something else then either the test name or test needs to change, so your comment is invalid when arguing something was wrong with the test name

Reply:
Also is this PR ready for review? This conversation is not resolved